src/android/index.jsx

```diff
@@ -299,44 +277,24 @@ class Android extends Component {
             <Grid item xs={6}>
               <PerfherderGraphContainer
                 timeDomain={timeDomain}
-                title="Unity WebGl"
-                series={[
-                  {
-                    label: 'Moto G5 (arm7)',
-                    filter: {
-                      and: [
-                        { missing: 'test' },
-                        {
-                          eq: {
-                            framework: 10,
-                            repo: 'mozilla-central',
-                            suite: 'raptor-unity-webgl-geckoview',
-                          },
-                        },
-                        {
-                          prefix: { platform: 'android-hw-g5-7-0-arm7-api-16' },
-                        },
-                      ],
-                    },
-                  },
-                  {
-                    label: 'Pixel 2 (ARM64)',
+                title="Unity WebGl Pixel 2"
+                series={selectFrom(COMBOS)
+                  .where({ suite: 'speedometer' })
+                  .map(({ browserLabel, filter: browserSuiteFilter }) => ({
+                    label: browserLabel,
                     filter: {
                       and: [
                         { missing: 'test' },
                         {
-                          eq: {
-                            framework: 10,
-                            options: 'opt',
```

Comment:
I think that, because you've removed the condition for options: 'opt' the chart shows both `pgo` and `opt` results, making it confusing like this:

![pgo opt](https://user-images.githubusercontent.com/51707553/76311580-f40adf80-62d9-11ea-808b-40f7b2642b1a.PNG)

Is it safe to say that we should only show results for `opt` platforms? @gmierz

Reply:
^ @davehunt

Reply:
@airimovici This looks confusing. Sorry for that. I just updated the series generator code. This is how it looks like now. `Speedometer`

![Capture](https://user-images.githubusercontent.com/30138596/76312385-f4ff3980-62f8-11ea-9ad8-a6960f7d8827.PNG)

Reply:
@ajinkabeer That looks better
👍 the same should be done for the Unity WebGl chart

Reply:
@airimovici Thanks for your quick response. I've also added the Power chart for respective platforms for you to take a look at. 

![Capture](https://user-images.githubusercontent.com/30138596/76313289-ab175300-62fa-11ea-8063-c699c342325f.PNG)

Reply:
As for the PowerSummary component, there is the platform that can be passed in props and have results for both platforms.

It's curios that there (in `src/power/config.js`) the `options: 'opt'` is commented on `p2-aarch64`

Reply:
Yes, I used the platform prop.
Yes, I just noticed that its hidden. Any specific reason for that you're aware of?
@airimovici

Reply:
I'm not sure about that. It definitely gives different results if the `options: 'opt'` filter is there:

Without `options: 'opt'`:
![speedometer without opt](https://user-images.githubusercontent.com/51707553/76314242-1a7f4980-62df-11ea-8fba-c04975f172ee.PNG)

With `options: 'opt'`:
![speedometer with opt](https://user-images.githubusercontent.com/51707553/76314262-20752a80-62df-11ea-8047-625cefec85a6.PNG)

Reply:
Yes, there's a huge difference between the two charts.

This is how the charts look now. `Unity WebGl Pixel 2` is not rendering. I have to figure why it doesn't render.

![Capture](https://user-images.githubusercontent.com/30138596/76314480-e61a8600-62fc-11ea-8c90-03442a6a97f6.PNG)

@airimovici

Reply:
From the current chart https://health.graphics/android, it seems like there is no data for the pixel 2 platform

Reply:
Oh I see. So how should we proceed then? 

@airimovici

Reply:
I think these changes should be included as you made them and see how we can proceed from there

Reply:
Alright, Thank you @airimovici. I have pushed the changes so that you can take a look at the preview.

Reply:
> 
> 
> I think that, because you've removed the condition for `options: 'opt'`, the chart shows both `pgo` and `opt` results, making it confusing like this:
> 
> ![pgo opt](https://user-images.githubusercontent.com/51707553/76311580-f40adf80-62d9-11ea-808b-40f7b2642b1a.PNG)
> 
> Is it safe to say that we should only show results for `opt` platforms? @gmierz

Sorry for the delay in replying here, I somehow missed all the mentions - please ping me on Riot if you're waiting for a response. @airimovici you should use `pgo` as much as possible since that's the build that we ship.